Extensions/Spine/spineruntimeobject-pixi-renderer.ts

```diff
@@ -0,0 +1,141 @@
+namespace gdjs {
+
+  const isSpine = (obj: any): obj is pixi_spine.Spine => obj instanceof pixi_spine.Spine;
+
+  export class SpineRuntimeObjectPixiRenderer {
+    private _object: gdjs.SpineRuntimeObject;
+    private _rendererObject!: pixi_spine.Spine | PIXI.Container;
+    private _isAnimationComplete = true;
+    private _spinePropertiesReplacer: Record<string, any> = { };
```

Comment:
replace either `Record<string, any>` to `{ [key: string]: any }` or just use a single boolean flag for instead of `this._spinePropertiesReplacer.autoUpdate`